Doc/library/sys.rst

```diff
@@ -281,7 +281,7 @@ always available.
    ``(type, value, traceback)``.  Their meaning is: *type* gets the type of the
    exception being handled (a subclass of :exc:`BaseException`); *value* gets
    the exception instance (an instance of the exception type); *traceback* gets
-   a traceback object (see the Reference Manual) which encapsulates the call
+   a traceback object (see :ref:`Reference Manual <traceback-objects>`) which encapsulates the call
```

Comment:
Instead of a link on `reference manual`, please add the link to `traceback object` and remove the parenthetical for `(see the Reference Manual`.  I think it will be cleaner that way.  Thanks!